### DISCUSSION TITLE
Cant upgrade neovim nightly version

### DISCUSSION THREAD

**Discussion by rwxmad** [neovim/neovim#25412] [General]
### Problem

Cant upgrade neovim nightly version via Homebrew

### Steps to reproduce

brew unlink neovim 
brew install neovim --HEAD

### Expected behavior

Installing success

### Neovim version (nvim -v)

v0.10.0-dev-772+g0211f889b

### Vim (not Nvim) behaves the same?

9.0.1544

### Operating system/version

macOs 14.0

### Terminal name/version

Alacritty 0.12.2

### $TERM environment variable

xterm-256color

### Installation

brew

**Comment by clason** [comment]
`brew install --HEAD` is not supported, not by us and not by homebrew (and is known to -- deliberately -- be broken).

If you must use a prerelease (because you want to help development), either build from source or use our nightly releases.

**Comment by rwxmad** [comment]
@clason I took the information from this source https://github.com/neovim/neovim/wiki/Installing-Neovim#pre-built-archives-1

**Reply by clason** [reply]
Thanks, I'll remove it.

**Comment by rwxmad** [comment]
@clason how can I install the nightly release, but not have to constantly run it from the download folder on macos?

**Reply by benjiwolff** [reply]
Feel free to use my tap [benjiwolff/homebrew-neovim-nightly](https://github.com/benjiwolff/homebrew-neovim-nightly).
